src/OAuthClient.js

```diff
@@ -315,9 +315,10 @@ OAuthClient.prototype.revoke = function revoke(params) {
 
     resolve(this.getTokenRequest(request));
   })
-    .then(({ request, ...authResponse }) => {
+    .then((res) => {
+      const { request, ...authResponse } = res.json ? res : null;
```

Comment:
This line will cause an error when trying to destructure `null`. 

![image](https://github.com/user-attachments/assets/4d0c9757-ce93-4bee-8edd-5801c07c708d)

> `Uncaught TypeError TypeError: Cannot destructure property 'request' of '(intermediate value)(intermediate value)(intermediate value)' as it is null.`

Reply:
Nice catch, silly mistake